cmd/minikube/cmd/start_flags.go

```diff
@@ -473,6 +473,11 @@ func getNetwork(driverName string) string {
 		return n
 	}
 	switch n {
+	case "vmnet-host", "vmnet-shared", "vmnet-bridged":
+		//TODO: check if QEMU v7.1+ version is installed
```

Comment:
Is there a required macOS version as well? Just wondering how far back the vmnet protocol is supported

Reply:
according to the [apple docs](https://developer.apple.com/documentation/vmnet): vmnet framework requires `macOS 10.10+`

Reply:
That's quite old, not sure if thats worth adding a version check for

Reply:
yeah, ten years now, so probably not